docs/Encyclopedia/pages/BWA-aln.md

```diff
@@ -0,0 +1,32 @@
+# BWA-aln
+
+## Description ##
+
+BWA-aln is an alignment workflow used in GDC miRNA-Seq data harmonization. BWA is an alignment algorithm that is used to align read groups to the reference genome.
+
+## Overview ##
+
+BWA-aln is a BWA based alignment workflow for miRNA-Seq. BWA-aln is used for aligning sequence reads to a reference genome for both normal and tumor tissues with BWA algorithm.
```

Comment:
```suggestion
See the Reference links below.
```

There was lots of repeated information, so I figure just pointing them to the links would be okay.

Reply:
changed